utils/check_deps.js

```diff
@@ -209,22 +219,16 @@ async function innerCheckDeps(root) {
 
   function allowExternalImport(importName, packageJSON) {
     // Only external imports are relevant. Files in src/web are bundled via webpack.
-    if (importName.startsWith('.') || importName.startsWith('@'))
+    if (importName.startsWith('.') || (importName.startsWith('@') && !importName.startsWith('@playwright/')))
       return true;
     if (peerDependencies.includes(importName))
       return true;
-    try {
```

Comment:
why did this go away?

Reply:
not needed anymore - we always resolve successfully in this code branch now